test/parallel/test-fs-write-buffer-large.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+// fs.write with length > INT32_MAX
+
+let buf;
+try {
+  buf = Buffer.allocUnsafe(0x7FFFFFFF + 1);
+} catch (e) {
+  // If the exception is not due to memory confinement then rethrow it.
+  if (e.message !== 'Array buffer allocation failed') throw (e);
+  common.skip('skipped due to memory requirements');
+}
```

Comment:
`0x7FFFFFFF + 1` causes an `ERR_INVALID_ARG_VALUE` on 32-bit platforms. So, you might want to use `skipIf32Bits` for this test.